lib/src/ui/home_page.dart

```diff
@@ -95,77 +78,12 @@ class _HomePageState extends State<HomePage> {
                   },
                 ),
         ],
-        flexibleSpace: Container(
-          decoration: BoxDecoration(
-            gradient: LinearGradient(
-              colors: [
-                Colors.red,
-                Colors.black,
-              ],
-              begin: Alignment.bottomRight,
-              end: Alignment.topLeft,
-            ),
-          ),
-        ),
+        flexibleSpace: MovieAppBarFlexibleSpaceStyle(),
```

Comment:
You can make this FlexibleSpace to receive a child widget in case that you want to show something on that space, make sense?

Reply:
It makes sense to me, in the future I could add some 'data' in that space. Thank you.